<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerfilWindow</class>
 <widget class="QDialog" name="PerfilWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonSair">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonVoltar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOla">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>20</width>
     <height>0</height>
    </size>
   </property>
   <property name="locale">
    <locale language="Portuguese" country="Brazil"/>
   </property>
   <property name="text">
    <string>wellcome</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelNome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNome">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelEmail">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEmail">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelIdade">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Idade</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIdade">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
